### ISSUE TITLE
修改网络之后的yolov8导出ncnn出现以下问题，该如何解决？

### ISSUE THREAD

**Issue by FJY1231** [ultralytics/ultralytics#15687]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

使用yolov8官方提供的代码进行pt-ncnn的转换，原始yolov8可以进行推理，但是使用我修改之后的网络无法推理，
官方代码：
from ultralytics import YOLO

Create a model
model = YOLO('ourmodule.pt')

Export the model to NCNN with arguments
model.export(format='ncnn', half=True, imgsz=640,batch=1)

出现以下结果：

D:\anancoda\envs\yolov8\python.exe E:\yolov8改进案例\ultralytics-20240713\ultralytics-main\nn.py
Ultralytics YOLOv8.2.50 🚀 Python-3.8.19 torch-2.2.1+cpu CPU (Intel Core(TM) i5-8250U 1.60GHz)
YOLOv8n-GhostHGNetV2-bifpn(c2f-star) summary (fused): 263 layers, 892893 parameters, 73856 gradients, 4.5 GFLOPs

PyTorch: starting from 'ourmodule.pt' with input shape (1, 3, 640, 640) BCHW and output shape(s) ((1, 14, 8400), ()) (2.4 MB)

TorchScript: starting export with torch 2.2.1+cpu...
TorchScript: export success ✅ 3.2s, saved as 'ourmodule.torchscript' (3.9 MB)

NCNN: starting export with NCNN 1.0.20240410...
NCNN: running 'E:\yolov8改进案例\ultralytics-20240713\ultralytics-main\ultralytics\pnnx.exe ourmodule.torchscript ncnnparam=ourmodule_ncnn_model\model.ncnn.param ncnnbin=ourmodule_ncnn_model\model.ncnn.bin ncnnpy=ourmodule_ncnn_model\model_ncnn.py pnnxparam=ourmodule_ncnn_model\model.pnnx.param pnnxbin=ourmodule_ncnn_model\model.pnnx.bin pnnxpy=ourmodule_ncnn_model\model_pnnx.py pnnxonnx=ourmodule_ncnn_model\model.pnnx.onnx fp16=1 device=cpu inputshape="[1, 3, 640, 640]"'
pnnxparam = ourmodule_ncnn_model\model.pnnx.param
pnnxbin = ourmodule_ncnn_model\model.pnnx.bin
pnnxpy = ourmodule_ncnn_model\model_pnnx.py
pnnxonnx = ourmodule_ncnn_model\model.pnnx.onnx
ncnnparam = ourmodule_ncnn_model\model.ncnn.param
ncnnbin = ourmodule_ncnn_model\model.ncnn.bin
ncnnpy = ourmodule_ncnn_model\model_ncnn.py
fp16 = 1
optlevel = 2
device = cpu
inputshape = [1,3,640,640]f32
inputshape2 =
customop =
moduleop =
############# pass_level0
inline module = torch.nn.modules.linear.Identity
inline module = ultralytics.nn.extra_modules.block.C2f_Star
inline module = ultralytics.nn.extra_modules.block.Fusion
inline module = ultralytics.nn.extra_modules.block.Ghost_HGBlock
inline module = ultralytics.nn.extra_modules.block.Star_Block
inline module = ultralytics.nn.extra_modules.head.Conv_GN
inline module = ultralytics.nn.extra_modules.head.DEConv_GN
inline module = ultralytics.nn.extra_modules.head.Detect_LSDECD
inline module = ultralytics.nn.extra_modules.head.Scale
inline module = ultralytics.nn.modules.block.DFL
inline module = ultralytics.nn.modules.block.HGStem
inline module = ultralytics.nn.modules.block.SPPF
inline module = ultralytics.nn.modules.conv.Conv
inline module = ultralytics.nn.modules.conv.DWConv
inline module = ultralytics.nn.modules.conv.GhostConv
inline module = torch.nn.modules.linear.Identity
inline module = ultralytics.nn.extra_modules.block.C2f_Star
inline module = ultralytics.nn.extra_modules.block.Fusion
inline module = ultralytics.nn.extra_modules.block.Ghost_HGBlock
inline module = ultralytics.nn.extra_modules.block.Star_Block
inline module = ultralytics.nn.extra_modules.head.Conv_GN
inline module = ultralytics.nn.extra_modules.head.DEConv_GN
inline module = ultralytics.nn.extra_modules.head.Detect_LSDECD
inline module = ultralytics.nn.extra_modules.head.Scale
inline module = ultralytics.nn.modules.block.DFL
inline module = ultralytics.nn.modules.block.HGStem
inline module = ultralytics.nn.modules.block.SPPF
inline module = ultralytics.nn.modules.conv.Conv
inline module = ultralytics.nn.modules.conv.DWConv
inline module = ultralytics.nn.modules.conv.GhostConv

############# pass_ncnn
stack along batch axis 0 is not supported
stack along batch axis 0 is not supported
stack along batch axis 0 is not supported
stack along batch axis 0 is not supported
stack along batch axis 0 is not supported
ignore Concat stack_0 param dim=0
ignore Concat stack_1 param dim=0
ignore Concat stack_2 param dim=0
ignore Concat stack_3 param dim=0
ignore Concat stack_4 param dim=0
NCNN: export success ✅ 2.6s, saved as 'ourmodule_ncnn_model' (2.2 MB)

Export complete (8.8s)
Results saved to E:\yolov8改进案例\ultralytics-20240713\ultralytics-main
Predict: yolo predict task=detect model=ourmodule_ncnn_model imgsz=640 half
Validate: yolo val task=detect model=ourmodule_ncnn_model imgsz=640 data=./ultralytics/cfg/datasets/coco128.yaml half
Visualize: https://netron.app/

进程已结束，退出代码为 0

**Comment by glenn-jocher** [comment]
@FJY1231 it appears that the issue may be related to the custom modifications made to your network. Please ensure that your custom layers and operations are compatible with the NCNN framework. If the problem persists, try exporting with the latest version of the Ultralytics package. For more detailed guidance, you can refer to our documentation at https://docs.ultralytics.com/integrations/ncnn/.
